### PR TITLE
Products

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Comment by MuskanChhatrasal** [comment]
Live Preview: https://flamboyant-allen-dfa0d4.netlify.app/pages/products

File: components/Products.css [hunk 1]
```diff
@@ -0,0 +1,171 @@
+.main-container {
+  display: flex;
+}
+.filters {
+  background-color: var(--primary-dark-color);
```
Comment (aayushjain-code) [inline components/Products.css:5]: Trry to add a little margin left in filter section

**Review by aayushjain-code — APPROVED** [review]
Looks good try to adjust bit of margins

File: components/Products.css [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+.footer .box-container .box .share a {
+  height: 4rem;
+  width: 4rem;
+  line-height: 4rem;
+  border-radius: 0.5rem;
+  font-size: 2rem;
+  color: var(--black);
+  margin-right: 0.2rem;
+  background: var(--background-color);
+  text-align: center;
```
Comment (ShekharDhangar) [inline components/Products.css:161]: use text-center as a helper class, it will helpful in many places

**Review by ShekharDhangar — APPROVED** [review]
try to use flex,flex-direction as helper class, rest is good

**Review by MayankKumar10 — APPROVED** [review]
Use form tag to write filter tag and to clear inputs add input reset so you can reset all seleted feilds

File: components/Products.css (hunk 2 above)
Comment (MuskanChhatrasal) [reply components/Products.css:161]: Okayy!! Will do that.

**Comment by MuskanChhatrasal** [comment]
Okayyy

File: components/Products.css (hunk 1 above)
Comment (MuskanChhatrasal) [reply components/Products.css:5]: Okayy!! Will do that.

**Pull request merged**